api/test/pact/pact-tests/rdLocation/getLocationsByServiceCode.spec.ts

```diff
@@ -0,0 +1,157 @@
+        'mrd_venue_id': somethingLike('MRD-CRT-0808'),
+        'service_url': somethingLike(''),
+        'fact_url': somethingLike('https://www.find-court-tribunal.service.gov.uk/courts/administrative-court')
+      }
+    ]
+  };
+
+  describe('get /getLocations}', () => {
+    const sandbox: sinon.SinonSandbox = sinon.createSandbox();
+    let next;
+
+    beforeEach(() => {
+      next = sandbox.spy();
+    });
+
+    before(async () => {
+      await pactSetUp.provider.setup();
+      const interaction = {
+        state: 'Search for locations',
+        uponReceiving: 'get list of court venues for given service code',
+        withRequest: {
+          method: 'GET',
+          path: '/refdata/location/court-venues/services',
+          query: `service_code=${serviceCode}`,
+          headers: {
+            'Authorization': 'Bearer someAuthorizationToken',
+            'ServiceAuthorization': 'Bearer someServiceAuthorizationToken',
+            'content-type': 'application/json'
+          }
+        },
+        willRespondWith: {
+          status: 200,
+          headers: {
+            'content-type': 'application/json'
+          },
+          body: RESPONSE_BODY
+        }
+      };
+      // @ts-ignore
+      pactSetUp.provider.addInteraction(interaction);
+    });
+
+    afterEach(() => {
+      sandbox.restore();
+      sinon.reset();
+    });
+
+    it('returns the correct response', async () => {
+      const configValues = getLocationsRefDataAPIOverrides(pactSetUp.provider.mockService.baseUrl);
+      sandbox.stub(config, 'get').callsFake((prop) => {
+        return configValues[prop];
+      });
+
+      const { getLocations } = requireReloaded('../../../../workAllocation/locationController.ts');
+
+      const req = mockReq({
+        headers: {
+          'Authorization': 'Bearer someAuthorizationToken',
+          'ServiceAuthorization': 'Bearer someServiceAuthorizationToken',
+          'content-type': 'application/json'
+        },
+        body: {
+          'locations': [
+            {
+              'id': '425094',
+              'userId': '9f17a6d6-8c89-42ff-8ebb-3bd3bac0c3a9',
+              'locationId': '425094',
+              'locationName': '',
+              'services': [
+                'CIVIL'
+              ]
+            }
+          ]
+        }
+
+      });
+      let returnedResponse = null;
+      const response = mockRes();
+      response.send = (ret) => {
+        returnedResponse = ret;
+      };
+
+      try {
+        await getLocations(req, response, next);
+        assertResponses(returnedResponse);
+        pactSetUp.provider.verify();
+        pactSetUp.provider.finalize();
+      } catch (err) {
+        console.log(err.stack);
+        pactSetUp.provider.verify();
+        pactSetUp.provider.finalize();
+        throw new Error(err);
+      }
+    });
+  });
+});
+
+function assertResponses(dto: any) {
+  console.log(JSON.stringify(dto));
```

Comment:
Looks good to me, just remove unnecessary logs from tests to clean up the pact test output